test/addons/argocd/test

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+from drenv import kubectl
+from drenv import commands
+
+
+def wait(cluster):
+    print("Waiting application to be healthy")
+    kubectl.wait(
+        "application",
+        "guestbook",
+        "--for=jsonpath={.status.health.status}=Healthy",
+        "--namespace=argocd",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+
+if len(sys.argv) != 4:
+    print(f"Usage: {sys.argv[0]} hub cluster1 cluster2")
+    sys.exit(1)
+
+hub = sys.argv[1]
+
+commands.run(
+    "argocd",
+    "app",
+    "create",
+    "guestbook",
+    "--repo",
+    "https://github.com/argoproj/argocd-example-apps.git",
+    "--path",
+    "guestbook",
+    "--dest-server",
+    "https://kubernetes.default.svc",
+    "--dest-namespace",
+    "argo-test",
+    "--sync-option",
+    "CreateNamespace=true",
+    "--sync-policy",
+    "automated",
+    "--kube-context",
+    hub,
+)
+
+wait(hub)
+
+commands.run("argocd", "app", "delete", "guestbook", "-y", "--kube-context", hub)
```

Comment:
Another issue, we are not waiting for the argocd deletion command. The test should wait until all cleanups are done before completing, so another test is not affected by this test side effects.

Reply:
Another issue - we prefer the long options in scripts, so the delete command should use "--yes" instead of "-y". This helps to create more clear scripts when using esoteric tool that no many people know. Not much better in this case but more consistent with other code.

Reply:
Updated code:
1. added a few more logs.
2. argocd-test namespace is deleted.
3. added --wait to the argocd app delete command.
4. --yes is used.